recite/admin.py

```diff
@@ -88,12 +102,13 @@ def save_model(self, request, obj, form, change):
         shutil.rmtree(temp_dir)
 
     def get_file_paths(self, temp_dir):
-        """Return file paths from temp_dir checking subdirectories"""
-        file_paths = []
+        """Return file paths dict from temp_dir checking subdirectories"""
+        file_paths = {}
```

Comment:
Let's make it `audio_paths[surah][ayah]` dictionary, please.
You will hide the initial format here and all other parts won't need to know about file naming conventions.